### DISCUSSION TITLE
I need to store large audio files, but can't

### DISCUSSION THREAD

**Discussion by Initialwave** [git-lfs/git-lfs#4588] [Q&A]
I followed the docs, but the files won't store. This is a unique problem I'd like help with. if you could reach out with a date/time that would be awesome

**Comment by bk2204** [comment]
Hey,

We're happy to try to help if you post the commands you've run and the exact output (copy-and-paste) of the results you've gotten, along with the output of `git lfs env`.  However, since this is an open source project, we don't set up individual appointments to meet with people; we work out in the open so that everyone can see what we're doing and learn from the experience.  I suspect your problem isn't completely unknown, and perhaps someone has seen this problem before and can jump in to offer assistance.

**Reply by Initialwave** [reply]
so following the docs, I ran git lfs install, and then I tracked my mp3s. After doing the general status, I did git add git.attributes, then the git add file.mp3, then commit, then push to my branch. it didn't accept the push ant threw a red error:
remote: warning: File sounds/rain.mp3 is 69.00 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
remote: error: Trace: 5de25635c822e5350b00e7af8df57d63b988aed8e8fb33d7b152696f04363c97
remote: error: See http://git.io/iEPt8g for more information.
remote: error: File sounds/beach.mp3 is 328.21 MB; this exceeds GitHub's file size limit of 100.00 MB
remote: error: File sounds/bonfire.mp3 is 173.14 MB; this exceeds GitHub's file size limit of 100.00 MB
remote: error: File sounds/rainforest.flac is 675.57 MB; this exceeds GitHub's file size limit of 100.00 MB
remote: error: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com.
To https://github.com/Initialwave/meditation-app.git
 ! [remote rejected] master -> master (pre-receive hook declined)
error: failed to push some refs to 'https://github.com/Initialwave/meditation-app.git'

**Reply by Initialwave** [reply]
so does GLF only have a max filesize of 100.00 MB?

**Reply by bk2204** [reply]
No, Git LFS doesn't have a maximum file size limit.  Your files aren't being tracked as LFS files, or you otherwise have these files as non-LFS files in your history.  If these files were added into the repository before you'd run `git lfs install` or `git lfs track`, that would explain why this is occurring, since you'd have large files already in the history.

What you can do now is to run `git lfs migrate import --everything --include "*.mp3,*.flac"`, which would rewrite your history such that the MP3 and FLAC files are properly LFS files throughout your history.

**Reply by Initialwave** [reply]
ok, I'll try that out and let you know

**Reply by Initialwave** [reply]
alright, I have it uploaded, but it added a master branch, didn't push to main, and won't allow any merge or pull requests from that branch now

**Reply by bk2204** [reply]
It should not create new branches.  You can just rename your branch with `git branch -m master main`, and then you'll need to force-push (that is, with the `-f` flag).  `git lfs migrate import` rewrites the entire history, so you'll need to recreate any pull requests based off the old history (probably with `git rebase --onto`).
